tests/unit/test_meshes/test_scikit_fem_submesh_3d.py

```diff
@@ -0,0 +1,183 @@
+import numpy as np
+import pytest
+
+import pybamm
+
+
+class TestScikitFemGenerator3D:
+    def test_box_mesh_creation(self):
+        try:
+            from pybamm.meshes.scikit_fem_submeshes_3d import ScikitFemGenerator3D
+        except ImportError:
+            pytest.skip("scikit-fem not available")
+
+        geometry = {
```

Comment:
can you update these test to have different lengths for each side just to catch any future bugs that flip axes?